src/Chapter13/Listing13.24.ConvertingExpressionTreeToSqlWhereClause.cs

```diff
@@ -1,10 +1,12 @@
 ﻿namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter13.Listing13_24
 {
     /*
-     * EXPRESSION TREE
-     *      persons.Where(person => person.Name.ToUpper() == "INIGO MONTOYA");
-     *      
-     * SQL WHERE CLAUSE
-     *      select * from Person where upper(Name) = 'INIGO MONTOYA';
+    #region INCLUDE
+    EXPRESSION TREE:
+    persons.Where(person => person.Name.ToUpper() == "INIGO MONTOYA");
+           
+    SQL WHERE CLAUSE:
+    select * from Person where upper(Name) = 'INIGO MONTOYA';
+    #endregion INCLUDE
```

Comment:
Should these lines be tabbed in to match the book?
```suggestion
    #region INCLUDE
    EXPRESSION TREE:
        persons.Where(person => person.Name.ToUpper() == "INIGO MONTOYA");
           
    SQL WHERE CLAUSE:
        select * from Person where upper(Name) = 'INIGO MONTOYA';
    #endregion INCLUDE
```